bucket/everything.json

```
@@ -14,16 +14,15 @@
         }
     },
     "pre_install": [
-        "if (!(Test-Path \"$persist_dir\\Everything.ini\")) { & \"$dir\\Everything.exe\" -install-config null }",
-        "if (!(Test-Path \"$persist_dir\\Everything.db\")) { New-Item \"$dir\\Everything.db\" | Out-Null }",
-        "if (Test-Path \"$persist_dir\\Bookmarks.csv\") {",
-        "    Copy-Item \"$persist_dir\\Bookmarks.csv\" \"$dir\"",
-        "} else {",
-        "    New-Item \"$dir\\Bookmarks.csv\" | Out-Null",
+        "ensure \"$persist_dir\" | Out-Null",
+        "if (Test-Path \"$persist_dir\\Everything.ini\") { Copy-Item \"$persist_dir\\Everything.ini\" \"$dir\" }",
+        "else { & \"$dir\\Everything.exe\" -install-config null }",
+        "'Everything.db','Bookmarks.csv' |% {",
+        "   if (Test-Path \"$persist_dir\\$_\") { Copy-Item \"$persist_dir\\$_\" \"$dir\" }",
         "}"
     ],
     "uninstaller": {
-        "script": "if (Test-Path \"$dir\\Bookmarks.csv\") { Copy-Item \"$dir\\Bookmarks.csv\" \"$persist_dir\" -Force }"
+        "script": "'Everything.ini','Everything.db','Bookmarks.csv' |% { Copy-Item \"$dir\\$_\" \"$persist_dir\" -ErrorAction SilentlyContinue -Force }"
```

Did you see any manifest with command aliases usage??
Do you think it is good idea to use aliases in readable scripts/definitions??

```suggestion
        "script": "Get-ChildItem \"$dir\\*\" -Include 'Everything.ini', 'Everything.db', 'Bookmarks.csv' | Copy-Item -Destination \"$persist_dir\" -ErrorAction SilentlyContinue -Force"
```

---

While I see your point I don't think it's a bad idea either.